translations/tiled_ru.ts

```diff
@@ -4733,8 +4733,8 @@ Please select specific format.</source>
     <message numerus="yes">
         <location filename="../src/tiled/flipmapobjects.cpp" line="+39"/>
         <source>Flip %n Object(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>Повернуть %n объект(ов)</numerusform>
```

Comment:
It seems Russian has space for three plural forms, but here you're only using one. Probably you could split it up and avoid the "(ов)" part, right?

The "(s)" is only part of the source text out of necessity, because there is only one version of source string.